src/utils/getFilteredPeople.ts

```diff
@@ -0,0 +1,63 @@
+import { Person } from '../types';
+
+export const getFilteredPeople = (
+  people: Person[],
+  sex: string | null,
+  query: string | null,
+  centuries: string[],
+  sortField: string | null,
+  order: string | null,
+): Person[] => {
+  let filteredPeople = [...people];
```

Comment:
The `filter` method returns a copy of the array, so you don't need to do it manually.